### DISCUSSION TITLE
Unable to Connect (You can try connecting via TCP)

### DISCUSSION THREAD

**Discussion by dayekaye** [mozilla/hubs#3365] [General Questions & Discussion] [answered]
I have some users who keep encountering this error message on certain rooms. The weird thing is that this is only happening on the rooms that are the smallest size, and not on the larger rooms. Those same users are able to access the larger rooms. Wondering if there is a fix or if this is something others have encountered? For some more context this room is based on a scene which has multiple rooms using the same scene. I am able to enter the room and so I am not able to reproduce the error myself but this is happening for users that haven't signed in. However, there are some users that haven't signed in that also do not encounter this error.

![image001](https://user-images.githubusercontent.com/6944431/99474824-5319fd00-291b-11eb-9aef-dfdd398987b4.jpg)

Some Console Messages
![image001](https://user-images.githubusercontent.com/6944431/99475024-a8eea500-291b-11eb-85ba-a489d8bdfc75.png)

**Comment by robinkwilson** [comment]
Very interesting. Your users are failing to connect due to an ERR_SSL_PROTOCOL_ERROR. 

There are a few guides if you search the web: `ERR_SSL_PROTOCOL_ERROR` on how to fix this error in the browser. Looks like fixes involve clearing caches + SSL state etc. I'm wondering if there are assets in the room that might be having issues loading? Very strange that they can connect to different rooms.

I'd love to hear what you find!

**Reply by vincentfretin** [reply]
This seems similar to https://github.com/mozilla/hubs/discussions/3091 where TLS traffic is not allowed on port 80 on some networks, but the fact that largest rooms work is strange.

**Reply by robinkwilson** [reply]
Exactly. If they can connect to any room at all, they're not running into the port 80 issue. This is room specific which leads me to believe that something in those rooms is causing issues.

**Reply by dayekaye** [reply]
I haven't changed anything in the scene, but one of the people that was encountering the issue is now able to have access to the room and no new users have encountered the issue. I Haven't been able to check with any of the 2-3 others that had this issue the other day though.
Would you be interested in any of the files I have in the scene, or screenshots from within Spoke?

**Reply by robinkwilson** [reply]
Quick questions:

1. Is this an Enterprise **multi-server** stack OR Personal/Enterprise **1 server** stack?
1. AWS or DigitalOcean deployed?

**Reply by dayekaye** [reply]
It is an Enterprise multi-stack server deployed through AWS

**Reply by robinkwilson** [reply]
How many multi-servers? 2x2?

**Comment by camelman90** [comment, marked as answer]
Why does it say "Using Janus SFU" (from hub.js) when it should go Dialogue?

Seems like it can't connect to the websocket, then falls back to old SFU somehow. 

I have a similar issue sometimes with my enterprise multi-server stack, but that is concerning all rooms will sometimes not connect because of the reticulum server (/meta) not responding. I haven't tested my own problem extensively, but it seems to be because I was using a 4G connection that was a bit wonky. Seems like it is a related issue, but the cause is different from mine (ssl error vs just not responding)

Have you tried terminating the EC2 instances and let them spin up new ones?

**Reply by robinkwilson** [reply]
In the code, we didn't change the webrtc service name so when it says "Janus" it means "Dialog".

+1 to @camelgod 's question on terminating the EC2 instances and spinning up new ones :)

**Reply by robinkwilson** [reply]
Marking as answer to get this out of our pipeline, when you have more information please "unmark as answer" so we can see it!
